library/src/main/res/values-v9/strings.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!-- Copyright 2012 Square, Inc. -->
 
-<resources>
-  <string name="month_name_format">LLLL yyyy</string>
+<resources xmlns:tools="http://schemas.android.com/tools">
+  <string name="month_name_format" tools:ignore="MissingTranslation">LLLL yyyy</string>
```

Comment:
I think I would prefer if you reverted this and downgraded the check to a warning in the `lintOptions` of the `build.gradle` instead.